backend/tracim_backend/error.py

```diff
@@ -76,6 +76,7 @@ class ErrorCode(IntEnum):
     USERNAME_ALREADY_EXIST_IN_DB = 2063
```

Comment:
Maybe rename this to `USERNAME_ALREADY_EXISTS_IN_DB` (or even `USERNAME_ALREADY_EXISTS`)?

Reply:
Done